dialogue/TownNPCs/Olive/InitialInteractions/AzuraBanana/AzuraBanana_Olive_First3.json

```diff
@@ -22,12 +22,20 @@
       "No sé que haría si algo le pasase a <c=pistachio>mi hermana pequeña</c>...",
       "¡Tened cuidado en vuestras aventuras!"
     ],
+    "pl": [
+      "No cóż, jestem zadowolona że wszytko u was jest w porządku.",
```

Comment:
You could change "jestem zadowolona" to "cieszę się" here too, because it sounds a bit more natural